src/prof.c

```diff
@@ -43,7 +43,7 @@ static counter_accum_t prof_idump_accumulated;
  * Initialized as opt_prof_active, and accessed via
  * prof_active_[gs]et{_unlocked,}().
  */
-bool prof_active;
+bool prof_active_state;
```

Comment:
This variable must be shadowed by something rather than shadowing something itself?

Reply:
Yep. Many functions have an input arg named prof_active. Since the global one usually shouldn’t be accessed anyway (it’s supposed to go through the getter), I renamed this one instead.